appium/tests/screenobjects/keys.screen.ts

```diff
@@ -64,20 +65,25 @@ class KeysScreen extends BaseScreen {
     return $(SELECTORS.BACK_BUTTON)
   }
 
-  checkKeysScreen = async () => {
-    //will add value verification for key later, need to create Api request for get key value
+  checkKeysScreen = async (keys?: KeyDetailInfo[]) => {
```

Comment:
I'd rather they are always passed explicitly, even if that means these usages need to be updated. It will make it more obvious exactly what is being tested.

Reply:
ok